docs/core/toolbar.component.md

```diff
@@ -80,6 +80,28 @@ You can divide groups of elements with a visual separator `<adf-toolbar-divider>
 </adf-toolbar>
 ```
 
+### Spacer
+
+You can provide a separate element with `adf-toolbar--spacer` class name
```

Comment:
Just to check... the "adf-toolbar--spacer" is supposed to have a double dash in it, right?

Reply:
Yes, that's the notation we use for certain elements